llvm/lib/Target/PowerPC/PPCISelDAGToDAG.cpp

```diff
@@ -510,7 +510,8 @@ SDNode *PPCDAGToDAGISel::getGlobalBaseReg() {
 }
 
 // Check if a SDValue has the toc-data attribute.
-static bool hasTocDataAttr(SDValue Val, unsigned PointerSize) {
+static bool hasTocDataAttr(SDValue Val, unsigned PointerSize,
+                           const PPCSubtarget *Subtarget) {
```

Comment:
I do not think we still need the function parameter `const PPCSubtarget *Subtarget` ?